src/vmm/src/devices/virtio/vsock/csm/connection.rs

```diff
@@ -374,24 +375,23 @@ where
             // A credit update from our peer is valid only in a state which allows data
             // transfer towards the peer.
             ConnState::Established | ConnState::PeerInit | ConnState::PeerClosed(false, _)
-                if pkt.op() == uapi::VSOCK_OP_CREDIT_UPDATE =>
+                if pkt.hdr.op() == uapi::VSOCK_OP_CREDIT_UPDATE =>
```

Comment:
nit: is it possible to add a unit test that covers this case?

Reply:
I am not very familiar with vsock unit tests. Also this is a bit outside the scope here.